src/main/java/org/intellij/sonar/persistence/Resource.java

```diff
@@ -0,0 +1,40 @@
+package org.intellij.sonar.persistence;
+
+public class Resource {
+    private String key;
+    private String name;
+    private String qualifier;
+
+    public Resource() {
+    }
+
+    public Resource(String key, String name, String longName, String scope, String qualifier, String description) {
+        this.key = key;
+        this.name = name;
+        this.qualifier = qualifier;
+    }
+
+    public String getKey() {
+        return key;
+    }
+
+    public void setKey(String key) {
+        this.key = key;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getQualifier() {
+        return qualifier;
+    }
+
+    public void setQualifier(String qualifier) {
+        this.qualifier = qualifier;
+    }
+}
```

Comment:
maybe also worse it to add equals and hashcode + toString()

Reply:
As you wish :-)